.github/workflows/specification-version-check.yml

```diff
@@ -0,0 +1,14 @@
+on:
+  schedule:
+    - cron: "0 13 * * *"
+  workflow_dispatch:
+name: Specification version check
+jobs:
+  # Get the latest TUF specification release and open an issue (if needed)
+  specification-bump-check:
+    permissions:
+      contents: read
+      issues: write
+    uses: theupdateframework/specification/.github/workflows/check-latest-spec-version.yml@master
+    with:
+      tuf-version: "v1.0.29" # Should be updated to the version the project supports either manually or extracted automatically. You can see how python-tuf did that as an example.
```

Comment:
I think we need to bring this down to the actual last version of the spec we specifically supported, which...I can't remember what it is. But we should change this before we merge.

Reply:
That makes sense 👍 Let me know which is it and I'll update the PR accordingly

Reply:
Sorry for the very long delay. Our current version of the spec is 1.0.18. If you can make this change in the PR, I will happily merge it.

Reply:
Thanks, @phenaproxima! Just did it 👍